functional_tests/steps/bundles.py

```diff
@@ -121,3 +126,166 @@ def the_user_can_see_the_bundle_details_with_creator(context: Context) -> None:
 def bundle_inspect_show(context: Context) -> None:
     expect(context.page.get_by_text("Created by")).to_be_visible()
     expect(context.page.get_by_text(context.bundle_creator.get_username())).not_to_be_visible()
+
+
+# Bundles UI Data Setup
+@given("there are {no_bundles} bundles")
+def multiple_bundles_create(context: Context, no_bundles: str) -> None:
+    context.bundles = []
+    if no_bundles.isdigit():
```

Comment:
The step will fail silently if `no_bundles` is not a digit.

I think it's worth failing if it's not:
```python
if not no_bundles.isdigit():
    raise ValueError(f"Invalid parameter: '{no_bundles}' must be a digit.")
```

Same problem on line 143, 155 and 170.